index.js

```diff
@@ -55,7 +55,8 @@ module.exports = class Localdrive {
         linkname: null,
         blob: null,
         metadata: null
-      }
+      },
+      mtime: st.mtimeMs
```

Comment:
We could consider call this `seq` but sounds wrong

Reply:
Example of `mtimeMs` => `1685669319303.709`

Reply:
ya prefer not to pretend